lib/internal/test_runner/harness.js

```diff
@@ -217,6 +220,17 @@ async function startSubtest(subtest) {
 }
 
 function runInParentContext(Factory) {
+  function validateTestOptions(options) {
+    ArrayPrototypeForEach(ObjectKeys(options), (key) => {
```

Comment:
wouldn’t this still allow me to pass `{ __proto__: whatever }` and it’d ignore those keys, but a Get later would still pull out their values? This might need to use for-in.

Reply:
Hi, I'm still trying to fully understand what you mean. `ArrayPrototypeForEach` already skips the `Object.prototype` values. Do you want me to make it also check those values too?

Reply:
what i mean is, `{ a, __proto__: { b } }` would be an options object where `options.a` and `options.b` would both work, but using ObjectKeys only would grab the `a`.

Reply:
So you want the validator to use a for-in loop that skips the `Object.prototype` keys?

Reply:
I'm not really sure. but if you really want to check for extra keys, you'd cover anything that's inherited except for those on Object.prototype.

Reply:
Alright then, I have an idea of how to do that.

Reply:
I've added the for-in loop that skips every `__proto__` values